api/app/Notifications/Forms/FormSubmissionNotification.php

```diff
@@ -51,14 +53,20 @@ public function toMail($notifiable)
             ->outputStringsOnly()
             ->useSignedUrlForFiles();
 
-        return (new MailMessage())
+        $mail = (new MailMessage())
             ->replyTo($this->getReplyToEmail($notifiable->routes['mail']))
             ->from($this->getFromEmail(), config('app.name'))
             ->subject('New form submission for "' . $this->event->form->title . '"')
             ->markdown('mail.form.submission-notification', [
                 'fields' => $formatter->getFieldsWithValue(),
                 'form' => $this->event->form,
             ]);
+
+        if ($this->mailer) {
```

Comment:
I don't think we need the if, since by default it uses the default mailer, we can always use this method without checking the condition

Reply:
And directly return like before